src/History.tsx

```diff
@@ -120,3 +85,70 @@ export class History extends React.Component<Props, {}> {
         );
     }
 }
+
+interface MessageProps {
+    activity: Activity;
+    autoscroll: () => void;
+    fromMe: boolean;
+    onClick?: React.MouseEventHandler<HTMLDivElement>;
+    selected: boolean;
+    showTimestamp: boolean;
+    store: ChatStore;
+    strings: Strings;
```

Comment:
Let's pass either `store` or `strings`, but not both. My preference is to pass in `store`, because that's the convention we use for most of our other React components. Then if you want, set a local variable to `store.getState().format.strings`. (If you do it this way, you can get rid of the `import { Strings } ... ` up top.